cmd/neutron_query_relayer/main.go

```diff
@@ -47,12 +48,37 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	wg := &sync.WaitGroup{}
 
+	// The storage has to be shared because of the LevelDB single process restriction.
+	storage, err := app.NewDefaultStorage(cfg, logger)
+	if err != nil {
+		logger.Fatal("Failed to loadRelayerStorage", zap.Error(err))
+	}
+	defer func(storage relay.Storage) {
+		err := storage.Close()
+		if err != nil {
+			logger.Fatal("Failed to close storage", zap.Error(err))
```

Comment:
Actually, I have just realized, we use `logger.Fatal` throughout code base. But doing so will not trigger any graceful shutdown logic, right?